samples/solidity/package-lock.json

```diff
@@ -1018,8 +1018,8 @@
       }
     },
     "@openzeppelin/contracts": {
-      "version": "4.5.0",
-      "resolved": "https://registry.npmjs.org/@openzeppelin/contracts/-/contracts-4.5.0.tgz",
+      "version": "4.7.1",
+      "resolved": "https://registry.npmjs.org/@openzeppelin/contracts/-/contracts-4.7.1.tgz",
       "integrity": "sha512-fdkzKPYMjrRiPK6K4y64e6GzULR7R7RwxSigHS8DDp7aWDeoReqsQI+cxHV1UuhAqX69L1lAaWDxenfP+xiqzA=="
```

Comment:
Typically the `package-lock.json` file is not meant to be edited directly. This change will not work because the SHA is not updated for the new package so `npm` will actually throw and error when it tries to install this.

Reply:
**Hi!** @nguyer 
Sorry for the delay and not being thorough, please change for patch the bellow : 
```json
"typescript": "^4.6.3"
},
"dependencies": {
"@openzeppelin/contracts": "^4.5.0"
"@openzeppelin/contracts": "^4.7.2"
}
}
```
upgrade a version to `"@openzeppelin/contracts": "^4.7.2"`